### ISSUE TITLE
very different columns height

### ISSUE THREAD

**Issue by alainib** [paulcollett/react-masonry-css#73]
hello. 
I'm trying this package, i get very different columns height because i display items with very different height. 
Some have title + little texte , some have long image more. 

<img width="1018" alt="Capture d’écran 2021-04-20 à 22 24 42" src="https://user-images.githubusercontent.com/7910081/115459348-3dd92f80-a227-11eb-8c99-c73bcbd193b2.png">

Is there a way to fix this ? 

I found a fix for now, sorting items with images first, but not sur it's a good idea because date is displayed 

Thanks

**Comment by yechs** [comment]
Yeah I had just the same problem and I think it is the intended behavior (at least for now). 

The authors comments in https://github.com/paulcollett/react-masonry-css/issues/64#issuecomment-695862802 that

> I'm aware of this behaviour and it's occurring because we don't actually handle this case, but i'm open to revisiting.
> 
> To implement it's slight performance trade off, as we need to calculate all the item heights and adjust the layout again, then again after each image has loaded. So for this plugin, avoiding all these re-layouts was actually the motivation behind making this plugin originally.. to be a fast and performant foremost.
